manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Is updating validatingwebhookconfiguration also in scope of this PR? Should we also update the code that uses admissionregistration v1beta1 package?

Reply:
Yes I have done that.